### PR TITLE
Update Rental Assistance Finder Tool

### PR DESCRIPTION
**Pull request by mistergone**
This Pull Request updates `@cfpb/rental-assistance-finder` to `0.6.0`. It also updates our local template to take advantage of new classes in the update.

[The 0.6.0 release can be found in the rental-assistance-finder repo.](https://github.com/cfpb/rental-assistance-finder/releases/tag/0.6.0)

[This release has been published on npm.](https://www.npmjs.com/package/@cfpb/rental-assistance-finder/v/0.6.0)

## Changes
- Updates `@cfpb/rental-assistance-finder` to `0.6.0`
- Updates `raf_tool.html` template with new classes.

## Screenshots
image

## Checklist
- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)

### CODE REVIEW

File: cfgov/cfgov/settings/base.py [hunk 1]
```diff
@@ -592,6 +592,12 @@
     "ROBOTS_TXT_SEARCH_GOV_ONLY": [("environment is", "beta")],
     # Whether to render auto-generated page translation links.
     "TRANSLATION_LINKS": [],
+    # RAFT Testing
+    "RAFT_TESTING": [
+        ("environment is", "beta"),
+        ("after date", "2023-01-22"),
+        ("before date", "2023-01-31"),
```
Comment (chosak) [inline cfgov/cfgov/settings/base.py:599]: @mistergone upon running the server after this change, Django outputs this warning:

```
?: (flags.E002) Flag RAFT_TESTING's 'after date' condition has an invalid value.
        HINT: Enter an ISO 8601 date representation.
?: (flags.E002) Flag RAFT_TESTING's 'before date' condition has an invalid value.
        HINT: Enter an ISO 8601 date representation.
```

The flags code [uses a Django function](https://github.com/cfpb/django-flags/blob/86ee7d9d7d169c5f66e3159843b2cec63a875ae6/flags/conditions/conditions.py#L87) to parse the string, which doesn't seem to recognize the date-only format, and returns null:

```
>>> from django.utils import dateparse
>>> dateparse.parse_datetime("2023-01-22")
>>>
```

If you include the time it seems to work properly:

```
>>> dateparse.parse_datetime("2023-01-22T00:00:00")
datetime.datetime(2023, 1, 22, 0, 0)
```

File: cfgov/cfgov/settings/base.py [hunk 2]
```diff
@@ -590,12 +590,7 @@
     "PATH_MATCHES_FOR_QUALTRICS": [],
     # Whether robots.txt should block all robots, except for Search.gov.
     "ROBOTS_TXT_SEARCH_GOV_ONLY": [("environment is", "beta")],
-    # RAFT Testing
-    "RAFT_TESTING": [
-        ("environment is", "beta"),
-        ("after date", "2023-01-22"),
-        ("before date", "2023-01-31"),
-    ],
+    # Whether to render auto-generated page translation links.
```
Comment (chosak) [inline cfgov/cfgov/settings/base.py:593]: This may have been introduced due to merge messiness?

```suggestion
```

Comment (mistergone) [reply cfgov/cfgov/settings/base.py:593]: Yes, there were some conflicts. I just resolved them, I think!

It seemed like removing that TRANSLATION flag altogether was the desired resolution, correct?

Comment (chosak) [reply cfgov/cfgov/settings/base.py:593]: Yes, correct, but also this comment should be removed as well, thank you!

**Review by brandon-rusk — APPROVED** [review]

**Pull request merged**